.github/workflows/iroha2-dev-pr-unstable.yml

```diff
@@ -12,7 +12,7 @@ on:
       # Not part of the workspace
       - "!wasm/**"
 
-concurrency: 
```

Comment:
@BAStos525 Please make sure that the tool you're using to write workflows cleans up whitespace. In YAML you can end up creating  very difficult to debug problems.